content/blog/case-weights/index.Rmd

```diff
@@ -0,0 +1,310 @@
+
+A framework like tidymodels enables all of these tools and the impact of case weights is more comprehensive. 
+
+Additionally, the type of case weights **and their intent** affect which of these operations should be affected. 
+
+For example, frequency weights should affect the model fit but also preprocessing steps as well as performance estimation. If the predictors require centering, a weighted mean should be used to appropriately make sure that the mean of that column is truly zero. Let's say that sensitivity and specificity estimates are required. The 2x2 table of observed and predicted results should have cell counts that reflect the case weights. If they did not, infrequently occurring data points have as much weight as the rows that have a high prevalence. 
+
+As a counter example, importance weights reflect the idea that they should influence _the model fitting procedure_. It wouldn't make sense to use a weighted mean to center a predictor; the weight shouldn't influence an unsupervised operation in the same way as the model fit. More critically, any holdout data set used to quantify model efficacy should reflect the data as seen in the wild (without the impact of the weights). 
+
+## How does tidymodels handle this? 
+
+We've decided to add some additional vector data types that allow users to describe the type of weights. These data types also help tidymodels functions know what the intent of the analysis should be. 
+
+In parsnip, the functions `frequency_weights()` and `importance_weights()` can be used to set the weights: 
+
+```{r vec-wts}
+# For the UC admissions data
+ucb$n <- frequency_weights(ucb$n)
+ucb
+
+# For a general vector of importance weights: 
+importance_weights(round(runif(10), 2))
+```
+
+The class of these objects tell packages like recipes and yardstick if their values should be used for preprocessing and performance metrics, respectively: 
+
+* Importance weights only affect the model fit and _supervised_ recipes steps. They are not used with yardstick functions
+
+* Frequency weights are used for all parts of the preprocessing, model fitting, and performance estimation operations. 
+
+Currently, these are the only classes implemented. We are doing a lot of reading on how the analysis of survey data should use case weights and how we can enable this and other data analysis use cases. [We'd love to hear from you](https://community.rstudio.com/) if you have expertise in this area. 
+
+## About resampling
+
+This is a topic that we are still unsure about. We definitively think that importance weights should not affect of the data are split or resampled. 
+
+Frequency weights are more complex. Suppose we are using 10-fold cross-validation with the logistic regression on the UCB admission data, should we
+
+* Have all the case weights be placed into either the modeling or holdout set? 
+* Fractionally, split the case weights into both the modeling and holdout data? 
```

Comment:
```suggestion
* Fractionally split the case weights into both the modeling and holdout data? 
```

I think I see what you were doing, but I also think this reads pretty well without the comma